language/evm/move-to-yul/src/functions.rs

```diff
@@ -1000,50 +999,23 @@ impl<'a> FunctionGenerator<'a> {
         for field_offs in layout.field_order.iter().take(layout.pointer_count) {
             let (byte_offs, ty) = layout.offsets.get(field_offs).unwrap();
             assert_eq!(byte_offs % 32, 0, "pointer fields are on word boundary");
-            if ty.is_vector() {
-                ctx.env
-                    .error(&self.parent.contract_loc, "vectors not yet implemented");
-                continue;
-            }
             ctx.emit_block(|| {
-                let hash = self.type_hash(ctx, ty);
-                // Add hash to names to avoid naming collision.
-                let linked_src_name = format!("$linked_src_{}", hash);
```

Comment:
Is there a rebase for your previous PR missing?

Reply:
Ah I moved part of this to `create_and_move_data_to_linked_storage` function instead, to enable more code reuse.